### ISSUE TITLE
I'm unable to use Hiren's boot cd

### ISSUE THREAD

**Issue by alcir** [aguslr/multibootusb#45]
I'm unable to use Hiren's boot CD.
I can boot it, ok, but I can't use any DOS program: no HBCD folder found.

**Comment by aguslr** [comment]
Hi @alcir,

Do you mean you cannot boot the "Dos programs" entry?

**Comment by alcir** [comment]
Yes, exactly. Where I'm wrong?

**Comment by aguslr** [comment]
If I understand correctly, when you are in GRUB's submenu for "Hiren's BootCD 15.2", you get this options:

```
Dos Programs
Windows Memory Diagnostic
MemTest86+
Offline NT/2000/XP/Vista/7 Password Changer
Kon-Boot
Seagate DiskWizard (Powered by Acronis Trueimage)
PLoP Boot Manager
Smart Boot Manager 3.7.1
Fix 'NTLDR is Missing'
Darik's Boot and Nuke - Hard Drive Eraser
Chainload GRUB4DOS
```

Then, when you highlight `Dos programs` and press ENTER, you get an error message saying "no HBCD folder found", right?

Can you provide an screenshot? How did you create the bootable USB drive? Did you follow the steps manually or did you use the script?

**Comment by alcir** [comment]
I select "Dos Programs"

![screenshot from 2017-01-13 10-23-03](https://cloud.githubusercontent.com/assets/3286123/21924690/7d3a2046-d97a-11e6-8791-187b64ac2489.png)

Then any of the DOS programs, i.e. "System Info Tools" -> "HWiNFO"

![screenshot from 2017-01-13 10-23-28](https://cloud.githubusercontent.com/assets/3286123/21924721/95694c14-d97a-11e6-9312-9a1ffab765e8.png)

![screenshot from 2017-01-13 10-23-40](https://cloud.githubusercontent.com/assets/3286123/21924742/ac1859b4-d97a-11e6-9fbf-2acdab764485.png)

After loading drivers, etc. I get the message "HBCD Folder not found"

![screenshot from 2017-01-13 10-23-49](https://cloud.githubusercontent.com/assets/3286123/21924755/b8d601c4-d97a-11e6-9a12-8a13d56c5d29.png)

No way if I select "Yes".
If I hit "No", I get a DOS prompt.

![screenshot from 2017-01-13 10-27-52](https://cloud.githubusercontent.com/assets/3286123/21924894/57f2cdc8-d97b-11e6-91c7-d728bf880c3c.png)

Then this is a snippet of the `dir` command

![screenshot from 2017-01-13 10-31-28](https://cloud.githubusercontent.com/assets/3286123/21924922/788b1e00-d97b-11e6-8970-7d19b36ad5f6.png)

**Comment by aguslr** [comment]
Hi @alcir,

It seems the option for `Dos programs` doesn't work when booted from an ISO file. I never really tried it before as I thought it worked since the menu loaded. Same thing with the option for `Mini Windows XP` which I did remove since it doesn't work.

I tried loading the ISO file with MEMDISK with the following:

```
menuentry "Load with MEMDISK" {
  bootoptions="iso raw vmalloc=600M"
  linux16 $prefix/memdisk $bootoptions
  initrd16 $isofile
}
```

However, even though `Mini Windows XP` does work this way, the option for `Dos programs` still fails with the message "Error 28: selected item cannot fit into memory". This option also requires more RAM (at least bigger than the ISO file).

I think the best solution is to delete the option for `Dos programs` and provide the alternative of loading the ISO with MEMDISK, since more tools are available in `Mini Windows XP`.
